### PR TITLE
test: add http-get connector test with mock endpoint

### PR DESCRIPTION
**Pull request by jhaeyaert**
## Description

This PR implements some tests with http-get and mock connectors.

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-axinofzjki.chromatic.com)

### CODE REVIEW

File: gravitee-apim-integration-tests/src/test/java/io/gravitee/apim/integration/tests/messages/httpget/HttpGetEntrypointMockEndpointIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+
+    @Override
+    public void configureEntrypoints(Map<String, EntrypointConnectorPlugin<?, ?>> entrypoints) {
+        entrypoints.putIfAbsent("http-get", EntrypointBuilder.build("http-get", HttpGetEntrypointConnectorFactory.class));
+    }
+
+    @Override
+    public void configurePolicies(Map<String, PolicyPlugin> policies) {
+        super.configurePolicies(policies);
+        policies.put("latency", PolicyBuilder.build("latency", LatencyPolicy.class, LatencyPolicy.LatencyConfiguration.class));
```
Comment (ytvnr) [inline gravitee-apim-integration-tests/src/test/java/io/gravitee/apim/integration/tests/messages/httpget/HttpGetEntrypointMockEndpointIntegrationTest.java:82]: just to confirm it's ok for you, this one is a fake policy, not the real latency one

File: gravitee-apim-integration-tests/src/test/java/io/gravitee/apim/integration/tests/messages/httpget/HttpGetEntrypointMockEndpointIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+            .assertValue(body -> {
+                assertThat(body.getString("message")).isEqualTo("Request timeout");
+                assertThat(body.getInteger("http_status_code")).isEqualTo(504);
+                return true;
+            })
+            .assertComplete();
+    }
+
+    @NonNull
+    private Flowable<JsonObject> extractMessages(Buffer body) {
```
Comment (ytvnr) [inline gravitee-apim-integration-tests/src/test/java/io/gravitee/apim/integration/tests/messages/httpget/HttpGetEntrypointMockEndpointIntegrationTest.java:223]:
```suggestion
    private Flowable<JsonObject> extractJsonMessages(Buffer body) {
```

**Review by ytvnr — COMMENTED** [review]
looks ok to me

File: gravitee-apim-integration-tests/src/test/java/io/gravitee/apim/integration/tests/messages/httpget/HttpGetEntrypointMockEndpointIntegrationTest.java (hunk 1 above)
Comment (jhaeyaert) [reply gravitee-apim-integration-tests/src/test/java/io/gravitee/apim/integration/tests/messages/httpget/HttpGetEntrypointMockEndpointIntegrationTest.java:82]: Yep, as long as it applies a delay it is ok👌

File: gravitee-apim-integration-tests/src/test/java/io/gravitee/apim/integration/tests/messages/httpget/HttpGetEntrypointMockEndpointIntegrationTest.java (hunk 2 above)
Comment (jhaeyaert) [reply gravitee-apim-integration-tests/src/test/java/io/gravitee/apim/integration/tests/messages/httpget/HttpGetEntrypointMockEndpointIntegrationTest.java:223]: I just wanted to keep the same name I used on other tests by default.

**Review by ytvnr — APPROVED** [review]

**Pull request merged**